libtenzir/builtins/contexts/geoip.cpp

```diff
@@ -471,8 +473,10 @@ class ctx final : public virtual context {
       co_yield slice;
     }
     // Dump all remaining entries that did not reach the size limit.
-    co_yield current_dump.builder.finish_assert_one_slice(
-      fmt::format("tenzir.{}.info", context_type()));
+    for (const auto& slice : current_dump.builder.finish_as_table_slice(
+           fmt::format("tenzir.{}.info", context_type()))) {
+      co_yield slice;
```

Comment:
Nitpick: In all hunks modified in this PR it should be `auto&& slice` followed by a `co_yield std::move(slice)`.